content/exception-filters.md

```diff
@@ -182,6 +182,56 @@ export class HttpExceptionFilter {
 
 > info **Hint** All exception filters should implement the generic `ExceptionFilter<T>` interface. This requires you to provide the `catch(exception: T, host: ArgumentsHost)` method with its indicated signature. `T` indicates the type of the exception.
 
+you can use `response.send()` instead of `response.json()` when using `platform-fastify`, and use `import { FastifyReply as Response, FastifyRequest as Request } from 'fastify';` instead of `import { Request, Response } from 'express';`
```

Comment:
With the way Dgeni (our markdown to angular component generator) works, you can't have `{` or `}` in the direct text (i.e. not a codeblock) without properly escaping it. If  recall, you have to do `{{ '{' }}` and `{{ '}' }}` to add them.

I think rather  we can re-write this sentence to be something like

```suggestion
If you are using `@nestjs/platform-fastify` you can use `response.send()` instead of `response.json()`. Don't forget to import the correct types from `fastify`.
```

and then have the example below to show the import of the types and .usage